modules/sde-core/src/test/java/org/eclipse/tractusx/sde/core/controller/SubmodelControllerTest.java

```diff
@@ -0,0 +1,70 @@
+package org.eclipse.tractusx.sde.core.controller;
```

Comment:
I assume you should also add license header to test java files.

You really need to configure your ide properly to handle the license header stuff automatically.